mozregression/bisector.py

```diff
@@ -58,6 +59,9 @@ def _print_progress(self, new_data):
         """
         raise NotImplementedError
 
+    def _reverse_if_hunt_fix(self, v1, v2):
```

Comment:
Since we're using this for more than just versions, let's call the variables str1 and str2